base/process.jl

```diff
@@ -69,7 +69,8 @@ function _uv_hook_close(proc::Process)
     nothing
 end
 
-const SpawnIOs = Vector{Any} # convenience name for readability
+const SpawnIO  = Redirectable
```

Comment:
This is a somewhat complicated union, which makes it non-ideal for asserting as the type. I think it also may be mostly deprecated, and mostly only present to help identify `FileRedirect` calls from other redirection calls.